core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -229,6 +229,15 @@ private synchronized void initContext(ClientContext ctx,
     mClosed.set(false);
     mMasterClientContext = MasterClientContext.newBuilder(ctx)
         .setMasterInquireClient(masterInquireClient).build();
+    try {
+      InetSocketAddress masterAddr = masterInquireClient.getPrimaryRpcAddress();
+      mMasterClientContext.loadConf(masterAddr, true, true);
+    } catch (UnavailableException e) {
+      LOG.error("Failed to get master address during initialization", e);
+    } catch (AlluxioStatusException ae) {
+      LOG.error("Failed to load configuration from "
+          + "meta master during initialization", ae);
+    }
```

Comment:
Should we only do this if metrics are enabled?

Reply:
For the safest change, we should only do this if metrics are enabled.
Update client configuration beforehand in all cases may help acquire and work with block worker client and file system master client.